README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/jru/include-what-you-use.svg?branch=master)](https://travis-ci.org/jru/include-what-you-use)
```

Comment:
Reminder for myself: this needs to be updated before we merge, to point to an "official" IWYU build.

Reply:
I can do that too!